src/Relude/Lifted/Terminal.hs

```diff
@@ -23,7 +23,6 @@ import Relude.String.Reexport (String, Text)
 import qualified Data.Text.IO as TIO
 import qualified System.IO as IO (print, putStr, putStrLn)
 
-
```

Comment:
This empty line should be here. According to our style guide, imports should be separated from the module definitions with 2 empty lines.

Reply:
cool, not sure why my IDE didn't auto format it. Will fix 😄